packages/web3-utils/test/unit/promise_helpers.test.ts

```diff
@@ -0,0 +1,54 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { waitWithTimeout, rejectIfTimeout } from '../../src/promise_helpers';
+
+const asyncHelper = async () => {
+	await new Promise(resolve => {
+		setTimeout(() => {
+			resolve('resolved');
+		}, 200);
+	});
+};
```

Comment:
```suggestion
```

Reply:
Related to below [suggested change](https://github.com/web3/web3.js/pull/5476/files/9869358cf0a5d128dec8d81603b5be514bee04e2#r987126923)